network/nordvpn/nordvpn.SlackBuild

```diff
@@ -23,14 +23,16 @@
 #  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # Modified by Pouria Rezaei <Pouria.rz@outlook.com>
+# Patched by Peter Christy <pete.christy.uk@gmail.com>
+
 
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=nordvpn
 VERSION=${VERSION:-3.19.0}
 BUILD=${BUILD:-1}
 TAG=${TAG:-_SBo}
-PKGTYPE=${PKGTYPE:-tgz}
+PKGTYPE=${PKGTYPE:-txz}
```

Comment:
please keep tgz for SBo
you can override it using local environment variable on your system using /etc/profile